S21/rkarmuri/Main.java

```diff
@@ -0,0 +1,21 @@
+//Main Program to convert to lowercase
+package com.company;
+
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Scanner myWord = new Scanner(System.in);//Input word
+        String wordSample;
+        Scanner myLang = new Scanner(System.in);//Input language tag
+        String language;
+        System.out.println("Enter a word to convert to lower case:");
+        wordSample = myWord.nextLine();//Move to next line after entering the word
```

Comment:
Can we not divide the initialization and assignment of the variable? We can do this and it will make it more readable:
String wordSample = myWord.nextLine();

Reply:
Yes we can do that.